soc/espressif/esp32/default_appcpu.ld

```diff
@@ -332,38 +446,20 @@ SECTIONS
   #include <zephyr/linker/ram-end.ld>
   _image_ram_size += _iram_end - _iram_start;
 
-  ASSERT(((__bss_end - ORIGIN(dram0_0_seg)) <= LENGTH(dram0_0_seg)),
+  ASSERT(((__bss_end - ORIGIN(dram0_1_seg)) <= LENGTH(dram0_1_seg)),
           "DRAM segment data does not fit.")
 
-  SECTION_DATA_PROLOGUE(_NOINIT_SECTION_NAME, (NOLOAD),)
-  {
-    . = ALIGN (8);
-    *(.noinit)
-    *(.noinit.*)
-    . = ALIGN (8);
-  } GROUP_LINK_IN(RAMABLE_REGION_1)
-
-  _image_irom_start = LOADADDR(.flash.text);
-  _image_irom_size = LOADADDR(.flash.text) + SIZEOF(.flash.text) - _image_irom_start;
-  _image_irom_vaddr = ADDR(.flash.text);
+  /* --- END OF DRAM --- */
 
-  .flash.text : ALIGN(4)
-  {
-    _stext = .;
-    _text_start = ABSOLUTE(.);
+  /* --- START OF IROM --- */
```

Comment:
I think this comments with empty content is only to tell there isn't DROM neither IROM stuff? Perhaps this could also go.